bindings/ergo-lib-wasm/src/ergo_box.rs

```diff
@@ -131,10 +166,10 @@ impl ErgoBox {
             .map(Constant::from)
     }
 
-    // JSON representation
-    // pub fn to_json(&self) -> Result<JsValue, JsValue> {
-    //     JsValue::from_serde(&self.0).map_err(|e| JsValue::from_str(&format!("{}", e)))
-    // }
+    /// JSON representation
+    pub fn to_json(&self) -> Result<JsValue, JsValue> {
+        JsValue::from_serde(&self.0.clone()).map_err(|e| JsValue::from_str(&format!("{}", e)))
```

Comment:
FYI I added `.clone()` to all `to_json` functions because otherwise the `to_json` would free to underlying WASM object which would cause a crash if you used the object again after calling `to_json` on it (which wasn't obvious)